src/Tizen.Applications.Common/Tizen.Applications/AppComponentType.cs

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.Applications
+{
+    /// <summary>
+    /// Enumeration for application component type.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum AppComponentType
```

Comment:
```suggestion
    public enum ApplicationComponentType
```
Do you need `Component` keyword?

`ApplicationType` Is it wrong?

Reply:
Because application type and application component type is different, so Component keyword is needed.

Reply:
Could you explain which is different?

Reply:
https://github.com/Samsung/TizenFX/blob/4173d1fbc061092b86c956d7237daeed3f5b24ac/src/Tizen.Applications.Common/Tizen.Applications/ApplicationType.cs#L23

Why don't you use this type?